app-emulation/vagrant/vagrant-2.2.10.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+# Drop ruby27: Many dependencies do not support it
+# USE_RUBY="ruby25 ruby26 ruby27"
+USE_RUBY="ruby25 ruby26"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="https://vagrantup.com/"
+SRC_URI="https://github.com/hashicorp/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox"
+RESTRICT="test"
+
+RDEPEND="${RDEPEND}
+	app-arch/libarchive
+	net-misc/curl
+	virtualbox? ( || ( app-emulation/virtualbox app-emulation/virtualbox-bin ) )"
+
+ruby_add_rdepend "
+	=dev-ruby/bcrypt_pbkdf-1.0*
+	=dev-ruby/childprocess-4.0*
+	>=dev-ruby/ed25519-1.2.4 <dev-ruby/ed25519-1.3
+	=dev-ruby/erubis-2.7*
+	>=dev-ruby/hashicorp-checkpoint-0.1.5 <dev-ruby/hashicorp-checkpoint-0.2
+	>=dev-ruby/i18n-1.8:1 <dev-ruby/i18n-2:1
+	>=dev-ruby/listen-3.1 <dev-ruby/listen-4
+	>=dev-ruby/log4r-1.1.9 <dev-ruby/log4r-1.1.11
+	>=dev-ruby/mime-0.4.4:* <dev-ruby/mime-0.5:*
+	=dev-ruby/net-ssh-6*:*
+	=dev-ruby/net-sftp-3*
+	=dev-ruby/net-scp-1.2*
```

Comment:
This list looks good, but these 3 are sorted the wrong way around. scp < sftp < ssh.